.gitignore

```diff
@@ -6,3 +6,5 @@
 /.pytest_cache
 /.tox
 /venv*
+/build
+/dist
```

Comment:
also looks like your other commit snuck in here: https://github.com/asottile/babi/pull/10

Reply:
Ah crap. Erm, how do I remove that?

Reply:
I think this is resolved now.

Reply:
you could rebase it out, `git rebase -i origin/master` and then delete the commits you don't want